Actitime/src/test/java/com/actimind/actitime/testcases/TC002.java

```diff
@@ -0,0 +1,33 @@
+package com.actimind.actitime.testcases;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import com.actimind.actitime.generic.ExcelLibrary;
+import com.actimind.actitime.pages.TasksPage;
+import com.actimind.actitime.pages.TimeTrackPage;
+
+public class TC002 extends BaseTest {
+	String customerName;
+	@Test
+	public void testCreateTask() {
+		
+		String option = ExcelLibrary.getStringData(XL_PATH, "TC001", 1, 0);
+		customerName = ExcelLibrary.getStringData(XL_PATH, "TC001", 1, 1);
+		String projectName = ExcelLibrary.getStringData(XL_PATH, "TC001", 1, 2);
+		String taskName = ExcelLibrary.getStringData(XL_PATH, "TC001", 1, 3);
+		int estimateHours = (int)(double)ExcelLibrary.getNumericData(XL_PATH, "TC001", 1, 4);
+		int deadlineDay = (int)(double)ExcelLibrary.getNumericData(XL_PATH, "TC001", 1, 5);
+		
+		TimeTrackPage timeTrackPage = (TimeTrackPage) homePage.clickOnMenuLink("Time-Track");
```

Comment:
Why are you Not taking this "Time Track" from ExcelSheet? Pls add this as a test data

Reply:
Oh, I missed it. Will fix it. Thanks